content/en/checkout/reference.md

```diff
@@ -119,13 +119,25 @@ An object with the following properties:
 - 429 Too Many Requests
 - 500 Internal Error
 
+#### Return value
+
+Returns a JSON Object (referred to as a *Checkout Manifest*) with the following properties:
+
+<div class="ep-protocol-list">
+
+- `id`: `string` with the Checkout session id.
+- `session`: `string` with a session token to be used by the rest of the Checkout process.
+- `config`: *deprecated* `Object` with configuration properties.
+
+</div>
+
 ## SDK
 
 ### `startCheckout(manifest, [options])`
 
 #### Parameters:
 
-- `manifest`: The return object from the [checkout service](#checkout).
+- `manifest`: The return object from the [checkout service](#create-checkout-session).
 - `options`: An optional object containing any of the following properties:
```

Comment:
Should this optional be removed to?

Reply:
I decided to leave it because the function definition marks the optionals instead of the required (by using the brackets). The text just reinforces that in case the user is not familiar with this common syntax.

Reply:
okay.